update_shaderc_sources.py

```diff
@@ -34,7 +34,7 @@
 VERBOSE = True
 
 
-def command_output(cmd, directory):
+def command_output(cmd, directory, fail_ok=False):
```

Comment:
Nit:  I would have added phrased this in the positive sense:  "muss_pass" option that defaults to True.

Not blocking though.